bot/exts/moderation/slowmode.py

```diff
@@ -44,16 +44,19 @@ async def get_slowmode(self, ctx: Context, channel: Optional[TextChannel]) -> No
         await ctx.send(f'The slowmode delay for {channel.mention} is {humanized_delay}.')
 
     @slowmode_group.command(name='set', aliases=['s'])
-    async def set_slowmode(self, ctx: Context, channel: Optional[TextChannel], delay: DurationDelta) -> None:
+    async def set_slowmode(self, ctx: Context, channel: Optional[TextChannel],
+                           delay: Union[DurationDelta, Literal["0s", "0seconds"]]) -> None:
```

Comment:
```suggestion
    async def set_slowmode(
        self,
        ctx: Context,
        channel: Optional[TextChannel],
        delay : Union[DurationDelta, Literal["0s", "0seconds"]],
    ) -> None:
```
To be consistent with how we format other function signatures in the bot